firstapp/app/src/androidTest/java/com/example/firstapp/ExampleInstrumentedTest.java

```diff
@@ -42,7 +50,7 @@ public void useAppContext() {
 
     @Test
     public void searchForTargetNumber() throws InterruptedException {
-        device = UtilityMethods.launchApp(BASIC_SAMPLE_PACKAGE, TIMEOUT);
+        UtilityMethods.launchApp(device, BASIC_SAMPLE_PACKAGE, TIMEOUT);
```

Comment:
Should be a more descriptive name than UtilityMethods.  Maybe something like ShellUtility.

Reply:
why shell?

Reply:
Just a name that I thought up in 5s :).  Any name that is more descriptive is fine with me.